keyboards/handwired/freoduo/rules.mk

```diff
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32u4
+# Bootloader selection
+BOOTLOADER = caterina
```

Comment:
```suggestion

# Bootloader selection
BOOTLOADER = caterina

```

Reply:
I don't understand this change. What changed?

Reply:
Added spacing around the block.

Reply:
The spacing. It adds a line between the MCU and bootloader section, adding a bit more readability.